wiki/People/Beatmap_Nomination_Group/en.md

```diff
@@ -9,33 +9,33 @@
 [Ym]: /wiki/shared/Insane-m.png "Yes"
 
 Beatmap Nomination Group
-=========================
+========================
 
-The **Beatmap Nomination Group**, more commonly referred to as BN, is a group in charge of [beatmap nomination](/wiki/Beatmaps).
+The **Beatmap Nomination Group**, more commonly referred to as the BNG, is a group in charge of [beatmap nomination](/wiki/Submissions/Beatmap_Ranking_Procedure). Individual members are commonly referred to as BNs, or Beatmap Nominators.
 This group has undergone many changes recently.
-As of August 22, 2014, they are no longer a part of the staff, revoking their chat moderation in-game, red name in-game, and on the forums.
-The most recent change was on February 1, 2015 when the team was changed from Beatmap Appreciation Team to Beatmap Nominator Group.
+As of 22. August 2014, they are no longer a part of the staff, revoking their chat moderation in-game, red name in-game, and on the forums.
+The most recent change was on 1. February 2015 when the team was changed from Beatmap Appreciation Team to Beatmap Nominator Group.
```

Comment:
Please read the ASG: 
```
Date Formatting

Since many regions use different date and time formats, the osu!wiki will only use this format: DD. MMMM YYYY.
```